robyn/__init__.py

```diff
@@ -467,6 +478,23 @@ def ALLOW_CORS(app: Robyn, origins: List[str]):
         app.add_response_header("Access-Control-Allow-Headers", "Content-Type, Authorization")
         app.add_response_header("Access-Control-Allow-Credentials", "true")
 
+        # https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request
+        @app.before_request()
```

Comment:
this is a very interesting way of implementation. The first time I am seeing this approach